### DISCUSSION TITLE
Pocketbase Deployed with Nginx 413 Request Entity Too Large

### DISCUSSION THREAD

**Discussion by kanchansapkota27** [pocketbase/pocketbase#883] [Q&A] [answered]
Tired uploading a pdf file from my localhost to pocketbase deployed on the server. Even when file size is set to greater than 5MB on uploading a 2MB pdf file it says something went wrong and looking at the console displays:

Status: 413 Request Entity Too Large

**Comment by ganigeorgiev** [comment, marked as answer]
This is probably due to some misconfiguration of the nginx proxy.

You could try changing the default [client_max_body_size](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) directive, eg.  `client_max_body_size 10M;`

**Reply by kanchansapkota27** [reply]
Thank you that worked just fine . It would be helpful to add that as a note in the documentation in the Deploy behind reverse proxy section itself in my opinion.
